fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowCatalogsStmt.java

```diff
@@ -22,25 +20,14 @@ public ShowCatalogsStmt() {
 
     }
 
-    @Override
-    public void analyze(Analyzer analyzer) throws AnalysisException {
-
-    }
-
     @Override
     public ShowResultSetMetaData getMetaData() {
         return META_DATA;
     }
 
     @Override
     public <R, C> R accept(AstVisitor<R, C> visitor, C context) {
-        return visitor.visitShowCatalogsStmt(this, context);
+        return visitor.visitShowCatalogsStatement(this, context);
     }
 
-    @Override
-    public String toSql() {
```

Comment:
why remove this?

Reply:
As mentioned in pr 5963, toSql() func is deleted in ShowCatalogsStmt.java , so UT in ShowCatalogsStmtTest is commented out.

Reply:
> why remove this?

![IMG_20220513_210752.png](https://user-images.githubusercontent.com/48429020/168290037-3172400d-9e75-4038-b00d-bbebe89dc522.png)